modin/pandas/groupby.py

```diff
@@ -83,6 +82,10 @@ def __init__(
         }
         self._kwargs.update(kwargs)
 
+    @property
+    def _index(self):
```

Comment:
Is it required to do the same for `columns`?

Reply:
No, currently none backend computes columns lazily.